### ISSUE TITLE
shuffle my own dataset

### ISSUE THREAD

**Issue by microchila** [ultralytics/ultralytics#15846]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Does ultralytics have an automatic go to shuffle operation during training?

**Comment by JastegSingh19** [comment]
Yes, during the training ultralytics shuffle before every epochs

**Comment by microchila** [comment]
@JastegSingh19  Is mosaic the data enhancement method that shuffles the image?
